### PR TITLE
Radio input bug

### PR DESCRIPTION
**Pull request by wytrych**
@incuna/js please review

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js bump

File: Gruntfile.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ module.exports = function (grunt) {
         uglify: _.extend({
             options: {
                 compress: {
-                    drop_console: true
+                    drop_console: false
```
Comment (henrahmagix) [inline Gruntfile.js]: Undo this change. Console calls should be removed from production code. `$log.log` won't, though

**Comment by henrahmagix** [comment]
:+1:

File: src/scripts/directives.js [hunk 2]
```diff
@@ -143,10 +143,15 @@
...
-                        value: scope.model
-                    };
-
+                    scope.localModel = '';
+                    var modelWatchDeregister = scope.$watch('model', function (value) {
+                        if (angular.isDefined(value)) {
+                            scope.localModel = {
+                                value: scope.model
+                            };
+                            modelWatchDeregister();
```
Comment (grahamgilchrist) [inline src/scripts/directives.js]: Why don't you want this watch to stay in place? If the parent attribute changes, you want to update the model inside the directive don't you?

Comment (henrahmagix) [inline src/scripts/directives.js]: I agree. However, object copy-by-reference makes it moot. But that isn't immediately visible or intuitive. Also, `scope.model` could be a primitive, like it is in the test.

Comment (wytrych) [inline src/scripts/directives.js]: Keeping this watch would make it blow up, as it would be recursively calling the second watch just below.

Comment (grahamgilchrist) [inline src/scripts/directives.js]: ah right I see. Sorry that second watch was chopped off in the PR so I didn't know what this was for. 
Looks good to me then, although perhaps a comment to explain exactly this will help us in future when we ask this same question :)

**Comment by wytrych** [comment]
Updated

**Comment by henrahmagix** [comment]
Odd that it's failing on the tests at lines 210 and 237

**Comment by wytrych** [comment]
Updated
